imports/plugins/core/versions/server/migrations/5_update_defaultRoles_to_groups.js

```diff
@@ -57,7 +57,7 @@ Migrations.add({
       ownerRoles = ownerRoles.concat(Reaction.defaultCustomerRoles);
       ownerRoles = _.uniq(ownerRoles);
 
-      const shopManagerRoles = ownerRoles.filter(role => role !== "owner");
+      const shopManagerRoles = ownerRoles.filter(role => role !== "owner" && role !== "admin");
```

Comment:
This is modifying an existing migration means it will not run for existing shops. 

Is this change critical enough that this migration must be run to adjust the admin roles for an existing shop? If so, then there will need to be a new migration that removes the admin role from the shop managers group.

Reply:
@mikemurray In my opinion yes, we should write a migration to adjust the admin roles for existing shops.
@impactmass What do you think about this?

Reply:
@Akarshit I agree it should be a separate migration to adjust it.